Mathmagician.Tests/FibonacciTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class FibonacciTests
+    {
+        [TestMethod]
+        public void GenerateInstanceOfFibonacci()
+        {
+            Fibonacci fibonacci = new Fibonacci();
+            Assert.IsNotNull(fibonacci);
+        }
+        [TestMethod]
+        public void GenerateANumber()
+        {
+            Fibonacci new_fib = new Fibonacci();
+            new_fib.GenerateFibonacci(1);
+            int fibToCheck = new_fib.sequence[0];
+            Assert.IsNotNull(fibToCheck);
+        }
+        [TestMethod]
+        public void WhatHappensIfYouEnterZero()
+        {
+            Fibonacci fibWithZero = new Fibonacci();
+            fibWithZero.GenerateFibonacci(0);
+            //the array already contains the first two numbers, 1 1
+            //this assert makes sure no new numbers were added.
+            Assert.IsTrue(fibWithZero.sequence.Count == 2);
```

Comment:
No, those numbers will not print. If 0 is fed into the method, it prints out a string and closes.